build/docker/Dockerfile

```diff
@@ -20,7 +22,7 @@ LABEL description="Honeydipper - an event-driven orchestration framework" \
 
 RUN apk add --no-cache ca-certificates git
 
-WORKDIR /opt/honeydipper/drivers/builtin
-COPY --from=build /go/bin/* ./
+COPY --from=build /build/honeydipper ./app/honeydipper
+
```

Comment:
@blazindragon We will need all the binaries, including the ones we call `drivers`. Just in case you haven't read it, see this https://github.com/honeydipper/honeydipper/blob/master/docs/DipperDaemon.png

Reply:
Ah my mistake, thanks for clarifying, will revert to building and copying all